src/commands/clear-account.js

```diff
@@ -0,0 +1,188 @@
+			const sts = new AWS.STS();
+			const caller = await sts.getCallerIdentity().promise();
+
+			const message = `You are about clear account [${
+				caller.Account
+			}] while using [${
+				profileName ? profileName : "unknown"
+			}] profile, are you sure?`;
+			const { clear } = await inquirer.prompt([
+				{
+					type: "confirm",
+					name: "clear",
+					message: message,
+					default: false
+				}
+			]);
+			if (clear) {
+				await this.clearEnvironment(AWS);
+			} else {
+				this.log("Not clearing environment");
+			}
+		}
+	}
+
+	_summary(results, singularName, hasRegion) {
+		let success = 0;
+		const failed = {};
+		results.forEach(val => {
+			const key = hasRegion ? `${val.name} [${val.region}]` : val.name;
+			if (val.status === "success") {
+				success++;
+			} else if (val.status === "fail") {
+				failed[key] = val.reason;
+			}
+		});
+		console.info("");
+		if (success > 0) {
+			this.log(`Successfully deleted ${success} ${singularName}(s)`.green.bold);
+		}
+		if (Object.keys(failed).length > 0) {
+			this.log(
+				`Failed deleting ${Object.keys(failed).length} ${singularName}(s)`.red
+					.bold
+			);
+			for (const [key, value] of Object.entries(failed)) {
+				this.log(`${key} - ${value}`.red);
+			}
+		}
+	}
+
+	async _resourceDeletion(countFunc, deleteAllFunc, singular, hasRegion) {
+		const count = await countFunc();
+		if (count > 0) {
+			this.log(`Deleting ${count} ${singular}(s)`);
+			const results = await deleteAllFunc();
+			this._summary(results, singular, hasRegion);
+		} else {
+			this.log(`No ${singular}(s) to delete. Skipping...`);
+		}
+	}
+
+	async _clearS3(AWS) {
+		this.log("S3".bold);
+		await this._resourceDeletion(
+			async () => {
+				return await getBucketCount(AWS);
+			},
+			async () => {
+				return await deleteAllBuckets(AWS);
+			},
+			"bucket",
+			false
+		);
+	}
+
+	async _clearCF(AWS) {
+		this.log("CloudFormation".bold);
+		await this._resourceDeletion(
+			async () => {
+				return await getAllStacksCount(AWS);
+			},
+			async () => {
+				return await deleteAllStacks(AWS);
+			},
+			"CF stack",
+			true
+		);
+	}
+
+	async _clearApiGw(AWS) {
+		this.log("API Gateway".bold);
+		await this._resourceDeletion(
+			async () => {
+				return await getAllApiGwCount(AWS);
+			},
+			async () => {
+				return await deleteAllApiGw(AWS);
+			},
+			"API Gateway",
+			true
+		);
+	}
+
+	async _clearRoles(AWS) {
+		this.log("IAM Roles".bold);
+		await this._resourceDeletion(
+			async () => {
+				return await getAllRolesCount(AWS);
+			},
+			async () => {
+				return await deleteAllRoles(AWS);
+			},
+			"role",
+			true
+		);
+	}
+
+	async _clearLambdas(AWS) {
+		this.log("Lambdas".bold);
+		await this._resourceDeletion(
+			async () => {
+				return await getAllLambdasCount(AWS);
+			},
+			async () => {
+				return await deleteAllLambdas(AWS);
+			},
+			"lambda",
+			true
+		);
+	}
+
+	async clearEnvironment(AWS) {
+		await this._clearS3(AWS);
+		console.info("");
```

Comment:
why the empty `console.info`?

Reply:
Because I'm using `process.stdout.write` to print `.` for each resource being deleted,  which does not add a new line in the end

Reply:
WDYT?

Reply:
I see, gotcha, sounds good!